Quake2-iOS/SavedGameViewController.swift

```diff
@@ -95,9 +86,11 @@ extension SavedGameViewController: UITableViewDataSource {
         let cell = tableView.dequeueReusableCell(withIdentifier: "cell")!
         
         cell.textLabel?.text = saves[indexPath.row]//.replacingOccurrences(of: ".svg", with: "")
-        #if os(tvOS)
-            cell.textLabel?.textColor = .black
-        #endif
+        cell.textLabel?.textColor = .white
+        cell.backgroundColor = .clear
+        cell.selectedBackgroundView = UIView.init(frame: .zero)
+        cell.selectedBackgroundView?.backgroundColor = UIColor.init(white: 1, alpha: 0.2)
+        cell.selectedBackgroundView?.layer.cornerRadius = 10
```

Comment:
Did you test these changes on tvOS? It appears as though the save games are unreadable now with white text on a white background that was fixed in #5.

Reply:
Guilty as charged! I hadn't tried it on tvOS. There we go; needed a storyboard update.